notebooks/official/automl/sdk_automl_forecasting_hierarchical_batch.ipynb

```diff
@@ -0,0 +1,1525 @@
+{
```

Comment:
<p>@reznitskii, I added some text that Andrew Leach and I put together for the hierarchical parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>